### PR TITLE
Qaseio v2 alpha

### PR DESCRIPTION
**Pull request by nl-kaidzen**
- Added autogenerated API Client;
- Updated imports;
- Updated `QaseApi` class;

### CODE REVIEW

File: qaseio/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "qaseio",
-  "version": "v1.5.0",
+  "version": "v2.0.0",
```
Comment (kamilsk) [inline qaseio/package.json:3]: please add `-alpha.0`

File: qaseio/src/api.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Qase.io API
```
Comment (kamilsk) [inline qaseio/src/api.ts:4]: please remove the same headers from everywhere

**Review by kamilsk — CHANGES_REQUESTED** [review]

File: qase-cucumberjs/package.json [hunk 3]
```diff
@@ -53,6 +53,6 @@
   "dependencies": {
     "chalk": "^4.1.0",
     "moment": "^2.29.1",
-    "qaseio": "^1.5.0"
+    "qaseio": "1.5.0"
```
[the remark this answers is not in the document]
Comment (nl-kaidzen) [reply qase-cucumberjs/package.json]: Fixed

File: qaseio/package.json (hunk 1 above)
Comment (nl-kaidzen) [reply qaseio/package.json]: Fixed

File: qaseio/src/api.ts (hunk 2 above)
Comment (nl-kaidzen) [reply qaseio/src/api.ts:4]: Done

**Review by kamilsk — APPROVED** [review]

**Pull request merged**